apiary.apib

```diff
@@ -719,7 +719,7 @@ API endpoints that Operation Code's Rails backend makes available to its React f
 
 + Response 400 (application/json)
 
-## Resource [/api/v1/resources/{resource_id}]
+## Resource | Delete [/api/v1/resources/{resource_id}]
```

Comment:
This would actually be the same route for both the `Show` or the `Delete` actions, similar to here:

https://github.com/OperationCode/operationcode_backend/blob/master/apiary.apib#L93